pytom/gui/guiStructures.py

```diff
@@ -5931,50 +5929,99 @@ def tab1UI(self, key, title='DataConversion'):
         self.items = [['', ] * columns, ] * rows
         w = 170
 
-        self.insert_label(parent,cstep=1,sizepolicy=self.sizePolicyB, width=400 )
+        self.insert_label(parent,cstep=0,sizepolicy=self.sizePolicyB, width=400 )
         self.insert_label_line_push(parent, 'Input file', wname=mode+'InputFile',width=w, initdir=self.tomogramfolder,
                                     tooltip='Select the tomogram file used for template matching.',
                                     filetype=['mrc', 'em', 'rec', 'st', 'log', 'txt', 'star', 'meta', 'xml'], mode='file')
         self.insert_label_line_push(parent,'Folder With Files', mode +'InputFolder',width=w,
                                     tooltip='Select the file where the sorted tiltimages are located.\n')
         self.insert_label_line_push(parent,'Output/Target Folder', mode +'TargetFolder',
-                                    'Select the folder where theoutput file(s) are saved.\n',width=w)
```

Comment:
These are no longer needed?

Reply:
Are you asking if:
- you think it is not needed it anymore? ans: it is an option under the `-t` flag of the script, hence it is still here)
- Why the `...Query` disappeared? ans: I reordered them based on the order in `convert.py` to make sure I kept track of all 17 options and now exist in line 5955-5958)

Reply:
It was about the Query, I probably didnt put the comment at the right spot. Good that you reordered them!